pkg/resource/notebook_instance/hooks.go

```diff
@@ -66,6 +66,10 @@ func (rm *resourceManager) customSetOutputDescribe(r *resource) error {
 				return err
 			}
 			r.ko.Status.StoppedByControllerMetadata = nil
+			// Notebook after rm.startNotebookInstance(r) is in pending state
+			// Manually change status since otherwise synced condition will be set to True as
+			// of a result Stopped status.
+			r.ko.Status.NotebookInstanceStatus = aws.String(svcsdk.NotebookInstanceStatusPending)
```

Comment:
Is this correct whenever the reconciler is doing a ReadOne call?

Reply:
The reconciler is doing a readOne call however it will get the latest status as Stopped which will cause the reconciler to no longer reconcile, so we will not be able to see the ensuing pending/in-service status that occur after the noteBook instance is started after being in the stopped state during this update codepath